Index: fluent-bit/init/systemd.in
===================================================================
--- fluent-bit.orig/init/systemd.in
+++ fluent-bit/init/systemd.in
@@ -3,12 +3,12 @@ Description=@FLB_PROG_NAME@
 Documentation=https://docs.fluentbit.io/manual/
 Requires=network.target
 After=network.target
+ConditionPathExists=@CMAKE_INSTALL_SYSCONFDIR@/@FLB_OUT_NAME@/@FLB_OUT_NAME@.conf
 
 [Service]
 Type=simple
-EnvironmentFile=-/etc/sysconfig/@FLB_OUT_NAME@
 EnvironmentFile=-/etc/default/@FLB_OUT_NAME@
-ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/@FLB_OUT_NAME@ -c /@CMAKE_INSTALL_SYSCONFDIR@/@FLB_OUT_NAME@/@FLB_OUT_NAME@.conf
+ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/@FLB_OUT_NAME@ -c @CMAKE_INSTALL_SYSCONFDIR@/@FLB_OUT_NAME@/@FLB_OUT_NAME@.yaml
 Restart=always
 
 [Install]
